ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -439,7 +454,22 @@ public int FindTargetForTouch(int reactTag, double touchX, double touchY)
             }
 
             var uiElement = view.As<UIElement>();
+#if WINDOWS_UWP
             var target = VisualTreeHelper.FindElementsInHostCoordinates(new Point(touchX, touchY), uiElement)
+#else
+            var sources = new List<DependencyObject>();
+            VisualTreeHelper.HitTest(
```

Comment:
Might be useful to put a TODO here to consider a pooled data structure.  You'll be allocating a lot of lists here in case of a touch heavy application